### DISCUSSION TITLE
Verify error: [403] Authorization failed

### DISCUSSION THREAD

**Discussion by Sturmflut92** [AsamK/signal-cli#1009] [Q&A] [answered]
hello,
i'm new and i try to run signal on my desktop pc without a smartphone.
i used this how-to: https://ctrl.alt.coop/en/post/signal-without-a-smartphone/
i registered my phone number and received the code via sms formatted like this:
**123456 (six digit number)
another code made out of letters and 1 or 2 numbers (not 100% sure, because my feature phone displays O and 0 the same...)**
i tried multiple variations but i always get the same error:
Verify error: [403] Authorization failed
this synthax doesn't work for me:
`$ bin/signal-cli -u YOUR_NUMBER verify VERIFICATIONCODE`
what am i doing wrong? how is the correct synthax to verify the phone number? the documentation says you only need a PIN, if you have set it manually before. i have a feeling that signal maybe changed that and always sends the user a PIN (the six digit number?) or are both lines part of the verificationcode?
would be awesome if someone could help me, thanks in advance and have a nice weekend!

**Comment by AsamK** [comment, marked as answer]
The VERIFICATIONCODE is the six digit number. The other code is just and android specific code for automatically filling in the code in the Signal-Android app and can be ignored.
The PIN is indeed only required if the number was locked with a PIN manually before, also an incorrect (or missing) PIN would show another more specific error message.

So use the six digit number for the verify command and be aware that the code is only valid for a limited time.

**Reply by Sturmflut92** [reply]
thanks for your support AsamK, your reply helped me a lot. your tool is genius!
